lib/view_component/base.rb

```diff
@@ -224,7 +232,7 @@ def controller
     # @return [ActionView::Base]
     def helpers
       raise HelpersCalledBeforeRenderError if view_context.nil?
-
+      raise StrictHelperError if ViewComponent::Base.config.strict_helpers_enabled
```

Comment:
I wonder if this feature might be less confusing if we just had it be `config.helpers_enabled` and defaulting it to true?

Reply:
@joelhawksley I think that's a good idea, when I have some time I will apply this feedback

Reply:
@joelhawksley I have changed the config to this, if you could review it again it would be a massive